src/marqo/client.py

```diff
@@ -192,6 +192,9 @@ def get_marqo(self):
         return self.http.get(path="")
 
     def health(self):
+        mq_logger.warning('The `client.health()` API has been deprecated and will be removed in '
+                          'Marqo 2.0.0. Use `client.index(index_name).health()` instead. '
+                          'Check `https://docs.marqo.ai/latest/API-Reference/indexes/` for more details.')
```

Comment:
What does this endpoint look like if you are using the cloud? Does it result in a JSON decode error? 

If so, perhaps it would be better to add some logic that checks if the URL corresponds to the cloud, and then give a more useful error (perhaps after catching the JSON decode error)